doc/decisions/decision_process.md

```diff
@@ -0,0 +1,137 @@
+# Decision Process
+
+## Problem
+
+Simply discussing in an issue and then doing an implementation is okay for non-substantial changes.
+But substantial decisions must be made transparently and in a participative way.
+
+## Constraints
+
+- All relevant information about decisions must be within Elektra's repository.
+- All decisions must go through at least two review rounds, with a merge in between.
+- At least two people need to be in favor of the decision in both rounds.
+- [Documentation guidelines](/doc/contrib/documentation.md) apply.
+- During the decision process, the PRs constantly get updated:
+  - Make changes as new commits to the pull request.
+  - Questions in the PRs are answered by updating the PR and answering with the commit id.
+  - As generally recommended in Elektra, do not squash after they are visible on the pull request.
+  - Rebase only if the decision was already accepted and has a merge conflict.
+- For reviewers:
+  - Prefer to directly give suggestions how to change sentences.
+  - General points should be asked in the root of "Conversation" and not at random sentences in the review.
+
+## Assumptions
+
+- People want to be informed about or even participate in what Elektra looks like in the future.
+- We will always be able to reach an consensus.
+  We don't need a vote or a benevolent dictatorship.
```

Comment:
How do we not need a vote, when the entire process is based on at least two people being in favor of the proposal in two seperate rounds? Isn't that already a vote?